<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UploadWidgetClass</class>
 <widget class="QWidget" name="UploadWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>660</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UploadWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Tab 1</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="DragDropLabel" name="select_file_lab">
         <property name="text">
          <string>选择mp3文件以读取文件信息</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Tab 2</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>604</width>
            <height>800</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>800</height>
           </size>
          </property>
          <widget class="QWidget" name="widget" native="true">
           <property name="geometry">
            <rect>
             <x>60</x>
             <y>10</y>
             <width>411</width>
             <height>61</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,9">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="label">
              <property name="styleSheet">
               <string notr="true">margin-top: 10px;</string>
              </property>
              <property name="text">
               <string>专辑封面</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
              </property>
              <property name="wordWrap">
               <bool>false</bool>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="DragDropLabel" name="select_cover_lab">
              <property name="text">
               <string>拖拽上传新的封面</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="widget_2" native="true">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>120</y>
             <width>185</width>
             <height>41</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>专辑名称</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="album_name_LineE"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="widget_3" native="true">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>230</y>
             <width>180</width>
             <height>41</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>发行日期</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="birthday_LineE"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="widget_4" native="true">
           <property name="geometry">
            <rect>
             <x>80</x>
             <y>280</y>
             <width>333</width>
             <height>111</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>专辑描述</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTextBrowser" name="album_desc"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="widget_5" native="true">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>400</y>
             <width>185</width>
             <height>41</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>作者名称</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="artist_LineE"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="widget_6" native="true">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>170</y>
             <width>185</width>
             <height>41</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>专辑序号</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="album_track_LineE"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="widget_7" native="true">
           <property name="geometry">
            <rect>
             <x>90</x>
             <y>70</y>
             <width>180</width>
             <height>41</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>歌曲名称</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="title_LineE"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="widget_8" native="true">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>560</y>
             <width>184</width>
             <height>41</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QPushButton" name="sure_btn">
              <property name="text">
               <string>确定</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="cancel_btn">
              <property name="text">
               <string>取消</string>
              </property>
              <property name="checkable">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="widget_9" native="true">
           <property name="geometry">
            <rect>
             <x>70</x>
             <y>480</y>
             <width>185</width>
             <height>41</height>
            </rect>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>总时长</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="duration_LineE">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DragDropLabel</class>
   <extends>QLabel</extends>
   <header>dragdroplabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
